app/flatpak-builtins-install.c

```diff
@@ -251,7 +251,7 @@ flatpak_builtin_install (int argc, char **argv, GCancellable *cancellable, GErro
   g_option_context_set_translation_domain (context, GETTEXT_PACKAGE);
 
   if (!flatpak_option_context_parse (context, options, &argc, &argv,
-                                     FLATPAK_BUILTIN_FLAG_STANDARD_DIRS,
+                                     FLATPAK_BUILTIN_FLAG_ALL_DIRS | FLATPAK_BUILTIN_FLAG_OPTIONAL_REPO,
```

Comment:
Do we want ALL_DIRS in `flatpak_complete_install` too? I know right now it has ONE_DIR but I think that's just an oversight from a33efa69c

Reply:
I was curious about this at some point - some of the completion implementations look a bit dubious to me. I have another pr with completion tests.

Reply:
I have another pr that fixes up mismatches between main and complete uses of dir flags in various commands.